deployment/config.yaml

```diff
@@ -109,3 +114,12 @@ cpu:
       dev: 100m
       staging: 500m
       prod: 500m
+
+bind_project:
+  bigquery: google.com:datcom-store-dev
+  bigtable: google.com:datcom-store-dev
+  branch_cache: google.com:datcom-store-dev
+  website_gcs: datcom-browser-staging # TODO(shifucun): use `gsutil acl ...`
```

Comment:
Should we also add a TODO to migrate that content to the new project?

Reply:
done